### ISSUE TITLE
cp950' codec can't decode byte 0xf0 in position 8324

### ISSUE THREAD

**Issue by tinlokkoo** [MaartenGr/KeyBERT#30]
```
      File "C:\Users\tinlok\AppData\Local\Temp\pip-req-build-hnmzzea8\setup.py", line 29, in <module>
        long_description = fh.read()
    UnicodeDecodeError: 'cp950' codec can't decode byte 0xf0 in position 8324: illegal multibyte sequence
```

please modify line 28 in setup.py to 
```
with open("README.md", "r", encoding='utf-8-sig') as fh:
    long_description = fh.read()
```

**Comment by MaartenGr** [comment]
Could you provide the exact steps you have taken to get to his issue? This helps me in finding out where the issue stems from.

**Comment by tinlokkoo** [comment]
just `pip install .` will cause this error. Some how, windows machine's default decoder is cp950. not utf-8. And for some file, there contains a ByteOrder Mark (BOM) which can be decode nicely with encoding='utf-8-sig'. This is a good practice to declear the file encoding if you know your encoding. So please change it.

**Comment by MaartenGr** [comment]
I will look into this and see if this can be fixed in the next version.

**Comment by MaartenGr** [comment]
This was fixed in the new v0.3 release (#32).
